ci.nix

```diff
@@ -1,4 +1,4 @@
-{ nospdk ? false, norust ? false, spdk_rel ? false }:
+{ nospdk ? false, norust ? false, spdk_rel ? false, asan ? false }:
```

Comment:
missing equivalent changes on shell.nix?

Reply:
shell.nix uses the system Rust, so it does not make sense to allow ASAN for it. ASAN forces a nightly Rust of a specific version.